core/storage/feedback/gae_models.py

```diff
@@ -73,9 +73,18 @@ class FeedbackThreadModel(base_models.BaseModel):
 
     @classmethod
     def generate_new_thread_id(cls, exploration_id):
-        """Generates a new thread id, unique within the exploration.
+        """Generates a new thread ID which is unique within the exploration.
 
-        Exploration ID + the generated thread ID is globally unique.
+        Args:
+            exploration_id: str, ID associated with each exploration
+
+        Returns:
+            thread_id: str, Thread ID is expressed as a sum of current time in
+            milliseconds and a random integer, or none if thread with same ID
+            already exists
+
+        Raises:
+            Exception: Thread with same ID already exists
```

Comment:
No, this is not the reason. Please read the function carefully and write the docstring more precisely.